### PR TITLE
Responsive fixes

### PR DESCRIPTION
**Pull request by emcelroy**

### CODE REVIEW

**Comment by scytacki** [comment]
@emcelroy did you test the touch changes on a Windows laptop and Chromebook that has both touch and pointer support?  
It seems like they might not work properly on those devices when someone is using the mouse.

File: src/library/components/resource-lightbox.js [hunk 1]
```diff
@@ -242,20 +242,32 @@ var ResourceLightbox = Component({
...
+    var license_attribution = '';
+    // don't provide suggested attribution for public domain resources
+    if (license.code !== 'CC-BY 4.0') {
+      license_attribution = p({}, 'Suggested attribution: ' + resource.name + ' by ',
+              a({href: 'https://concord.org/'}, 'The Concord Consortium'),
+              ' is licensed under ',
+              a({href: license.deed}, license.code),
+              '.'
+            );
+    }
```
Comment (scytacki) [inline src/library/components/resource-lightbox.js]: What about resources that CC doesn't own? We have a few of these in the portal. So if they choose anything other than CC-BY 4.0 we are going to suggest that concord should be attributed.

I suspect we are going to need to move this into the data model somehow.

Comment (scytacki) [reply src/library/components/resource-lightbox.js]: I suggest pulling this change out of this PR so we can get the rest of the changes merged in.

Comment (scytacki) [reply src/library/components/resource-lightbox.js]: The portal sends a 'credits' field that can be used for this. If it isn't set then we could set it to Concord Consortium.

**Comment by emcelroy** [comment]
@scytacki I think the license text changes are all set, as we discussed yesterday.

I'm not so sure about the changes for devices with both touch and pointer support. What I've done seems to work. Supporting touch-enabled devices with a mouse or trackpad is pretty tricky though and my solution seems overly complex. I'm not sure there's a better solution that doesn't require a complete reworking of the mouseover and mouse-click event handling though. What do you think? Is this a better option than disabling card hover states completely for all touch-enabled devices?

File: src/library/components/stem-finder-result.js [hunk 2]
```diff
@@ -53,16 +87,18 @@ var StemFinderResult = Component({
...
     // mount/unmount lightbox outside of homepage content
-    if (lightbox) {
+    if (lightbox && page_scrolling == false) {
       var resourceLightbox = ResourceLightbox({
         resource: this.props.resource,
         toggleLightbox: this.toggleLightbox
       });
       Lightbox.open(resourceLightbox);
-      _gaq.push(['_trackEvent','Home Page Resource Card','Click',this.props.resource.name]);
+      //_gaq.push(['_trackEvent','Home Page Resource Card','Click',this.props.resource.name]);
```
Comment (scytacki) [inline src/library/components/stem-finder-result.js]: Did you mean disable this analytics event?

Comment (dougmartin) [reply src/library/components/stem-finder-result.js]: No I didn't mean to leave it in.  I'll remove it.

**Pull request merged**